app/models/audit/event/authn.rb

```diff
@@ -81,8 +84,10 @@ def attempted_action
       def auth_stuctured_data
         { authenticator: @authenticator_name }.tap do |sd|
           sd[:service] = service_id if @service
+          sd[:user] = @success || Role[role_id: @role_id] ? @role_id : NOT_FOUND #Masking role if it doesn't exist to avoid audit pollution
```

Comment:
please move the comment to be before the line and not at its end. 
Also - add a space after the `#`